routers/api/v1/repo/pull.go

```diff
@@ -405,6 +405,18 @@ func EditPullRequest(ctx *context.APIContext, form api.EditPullRequestOption) {
 		}
 	}
 
+	if ctx.Repo.IsWriter() && (form.Labels != nil && len(form.Labels) > 0) {
```

Comment:
Iisn't `(form.Labels != nil && len(form.Labels) > 0)` testing the same thing?

Reply:
No, since form.Labels may be empty but not nil.

Reply:
Ah ok, thanks for clarifying!